ts/screens/profile/components/services/ManualConfigBottomSheet.tsx

```diff
@@ -23,24 +18,40 @@ export const useManualConfigBottomSheet = (onConfirm: () => void) => {
     present,
     bottomSheet: manualConfigBottomSheet,
     dismiss
-  } = useLegacyIOBottomSheetModal(
-    <ManualConfigConfirm />,
-    I18n.t("services.optIn.preferences.manualConfig.bottomSheet.title"),
-    350,
-    <FooterWithButtons
-      type={"TwoButtonsInlineHalf"}
-      leftButton={{
-        ...cancelButtonProps(() => dismiss()),
-        onPressWithGestureHandler: true
-      }}
-      rightButton={{
-        ...errorButtonProps(() => {
-          onConfirm();
-          dismiss();
-        }),
-        onPressWithGestureHandler: true
-      }}
-    />
+  } = useIOBottomSheetAutoresizableModal(
+    {
+      title: I18n.t(
+        "services.optIn.preferences.manualConfig.bottomSheet.title"
+      ),
+      component: <ManualConfigConfirm />,
+      fullScreen: true,
+      footer: (
+        <FooterWithButtons
+          type="TwoButtonsInlineHalf"
+          primary={{
+            type: "Outline",
+            buttonProps: {
+              label: I18n.t("global.buttons.cancel"),
+              onPress: () => dismiss(),
+              accessibilityLabel: I18n.t("global.buttons.cancel")
+            }
+          }}
+          secondary={{
+            type: "Solid",
+            buttonProps: {
+              color: "danger",
+              label: I18n.t("global.buttons.confirm"),
+              accessibilityLabel: I18n.t("global.buttons.confirm"),
+              onPress: () => {
+                onConfirm();
+                dismiss();
+              }
+            }
+          }}
+        />
+      )
+    },
+    250
```

Comment:
What is this number? Can we put it in a more readable constant? What do you think?

Reply:
This number is used by the hook, it is a snapPoint! However yes, I will put it in a variable!